bin/rtdev-build-releases.sh

```diff
@@ -81,9 +81,9 @@ kerl()
 
     echo " - Building Erlang $RELEASE (this could take a while)"
     # Use the Basho-patched version of Erlang 
-    if [ "$RELEASE" == "R15B01" ]; then
+    if [ ! -z $(echo "$RELEASE" | grep "R15B01") ]; then
         BUILD_CMD="./kerl build git git://github.com/basho/otp.git $R15_TAG $BUILDNAME"
-    elif [ "$RELEASE" == "R16B02" ]; then
+    elif [ ! -z $(echo "$RELEASE" | grep "R16B02") ]; then
```

Comment:
Now the bash operator for string equality is actually `-eq` but it still worked with `==`